interface/src/app/signature/signature.js

```diff
@@ -23,8 +23,39 @@ angular.module('adage.signature', [
   });
 }])
 
-.controller('SignatureCtrl', ['Signature', '$stateParams', '$log',
-  function SignatureController(Signature, $stateParams, $log) {
+.component('participationTypeSelector', {
```

Comment:
Do you think this component may be used in any other modules? If yes, it will be better to put in a separate module instead of in `adage.signature`.